cpmpy/solvers/TEMPLATE.py

```diff
@@ -92,12 +92,14 @@ def __init__(self, cpm_model=None, subsolver=None):
         super().__init__(name="TEMPLATE", cpm_model=cpm_model)
 
 
-    def solve(self, time_limit=None, **kwargs):
+    def solve(self, time_limit=None, display=None, **kwargs):
         """
             Call the TEMPLATE solver
 
             Arguments:
             - time_limit:  maximum solve time in seconds (float, optional)
+            - display: display: either a list of CPMpy expressions, OR a callback function, called with the variables after value-mapping
```

Comment:
makes it seem as if the "variables after value-mapping" would get passed to the display callback

Reply:
Ah hmm I see, I copy-pasted it from the `solveAll` docstring so maybe that one should be updated too? 
Feel free to propose an alternative that would make it more clear : )

Reply:
Updated this description to be more clear